oximeter/producer/src/lib.rs

```diff
@@ -180,7 +180,8 @@ pub async fn register(
     client
         .cpapi_producers_post(&server_info.into())
         .await
-        .map_err(|msg| Error::RegistrationError(msg.to_string()))
+        .map_err(|msg| Error::RegistrationError(msg.to_string()))?;
+    Ok(())
```

Comment:
yeah, this is an unrelated progenitor update that we'll catch on the next dependabot sync